hkm/templates/hkm/views/404_image.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'hkm/base.html' %}
+{% load i18n %}
+
+{% block main_content %}
+
+  <div class="status-page">
+    <img class="status-page__image" src="/static/hkm/img/no-results-image.jpg"></img>
```

Comment:
If this image is decorative, it should be marked so with an empty alt (`alt=""`). Otherwise it should have an alt description.